shell/components/Inactivity.vue

```diff
@@ -262,6 +238,22 @@ export default {
 
       checkCountdown();
     },
+    async checkBackendInactivity(currExpiresAt) {
```

Comment:
nit, if you're making functional changes (not just removing console logs) this can just be `async checkBackendInactivity()` again (it's called in two places with the same arg `this.checkBackendInactivity(this.expiresAt);`

Reply:
no other functional changes, no need to tweak